docs/wiki/EL/data-structures.md

```diff
@@ -186,7 +186,25 @@ The **World State Trie** is a living structure that evolves with each block, unl
 
 In summary, Ethereum's world state is a secure and verifiable representation of the current state of all accounts at a given block height.
 
-### TODO: Explain Storage Trie
+## Storage Trie
+
+In the previous section, we described how each account leaf in the **World State Trie** contains a `storageRoot`, which is the keccak-256 hash of the root node of a separate Merkle Patricia Trie, the **Storage Trie**. This trie is not embedded in the **World State Trie** but referenced via the `storageRoot`, enabling storage to be updated and proven independently while still contributing to the global state root.
+
+The **Storage Trie** represents a contract’s persistent state as a mapping of 256-bit storage slots indices (keys) to 256-bit RLP-encoded values. Like the World State Trie, it uses a secure key scheme where each slot index is hashed with keccak-256 before insertion.This prevents attackers from crafting keys that cause long traversal paths or highly unbalanced trie structures, which could otherwise be exploited for DOS attacks by inducing excessive computation during trie lookups or updates.
```

Comment:
Suggestion: Indicate the mapping of index -> value is sometimes called a storage slot. You have used the "storage slot" in the note below, so maybe define it here.